vue-components/src/components/Input.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+	<input
+		type="text"
+		:class="classes"
+	>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+
+/**
+ * This is an internal component, not to be confused with the TextInput system component, which is intended to be shared
+ * between system components such as TextInput and Lookup that have an input field which should look and behave
+ * identical.
+ */
+export default Vue.extend( {
+	name: 'Input',
+	props: {
+		feedbackType: {
+			type: String,
+			validator( value: string|null ): boolean {
+				return value === null ||
+					[ 'warning', 'error' ].includes( value );
+			},
+			default: null,
+		},
+	},
+
+	computed: {
+		classes(): string[] {
+			const classes = [ 'wikit-Input' ];
+
+			if ( this.feedbackType ) {
```

Comment:
does this ever resolve to false ?
AFAIK, typescript objects always resolve to true, even if they're empty.
are we checking here if feedbackType is empty or undefined ?

Reply:
`feedbackType` is a prop that is `null` by default (if not specified) or one of the strings specified in the prop validator. So yes, it can be falsy!